### PR TITLE
Inline Help: manually revise auto-generated descriptions for contextual help articles

### PR DESCRIPTION
**Pull request by lsinger**
The first iteration of contextual content added help articles and their descriptions as provided by the API. It was an improvement, but many of the auto-generated descriptions are less than ideal in the Inline Help context. This PR has revised titles and descriptions for many of the contextual help articles. 

To test: 
- Ponder whether the titles and descriptions are an improvement or at least not a deterioration of what we're currently showing.

### CODE REVIEW

**Comment by michelleweber** [comment]
Could you drop screen shots or plain text of what you need reviewed here? I can't see the link. Thanks much!

**Comment by lsinger** [comment]
@michelleweber I think the best way to review this would be to look at [the diff here](https://github.com/Automattic/wp-calypso/pull/24709/files) to see both the old and new versions side by side. We're mostly concerned with making sure things are a bit clearer in the new version and that there are no grave mistakes — given the volume I don't think we need to shot for perfection at this time. 

To see how these will be used, head to https://wordpress.com, click the question mark in the lower right, and then click the first suggested article ("Reader") — you'll then see this:

screen shot 2018-05-07 at 4 47 34 pm

**Comment by lsinger** [comment]
If you do the same not using WordPress.com but [this link](https://calypso.live/?branch=update/inline-help/revise-article-summaries) you'll get the updated version, but going through it in the UI is not advisable since the suggestions differ depending on the page you're on:

screen shot 2018-05-07 at 4 49 11 pm

**Comment by michelleweber** [comment]
Got it. I'm at a team meetup ATM, but will go through the full PR after dinner tonight.

File: client/blocks/inline-help/contextual-help.js [hunk 1]
```diff
@@ -554,115 +551,114 @@ const contextLinksForSection = {
...
+			title: 'The Reader: An Overview',
 			description:
-				'Read posts from all the sites you follow (even the ones that aren’t on WordPress.com), find great new reads, and keep...',
+				'Read posts from all the sites you follow — even ones that aren’t on WordPress.com! ' +
+				'Discover great new reads and keep track of your comments and replies in one convenient place.',
 		},
 		{
 			link: 'http://en.support.wordpress.com/following/',
-			title: 'Following Blogs',
+			title: 'Follow Blogs',
```
[the remark this answers is not in the document]
Comment (lsinger) [reply client/blocks/inline-help/contextual-help.js]: We use the non-gerund-form (not sure what the correct term would be) in most places already so I switched a few cases over for consistency.

File: client/blocks/inline-help/contextual-help.js [hunk 2]
```diff
@@ -40,129 +40,130 @@ const contextLinksForSection = {
 	stats: [
 		{
 			link: 'http://en.support.wordpress.com/stats/',
-			title: 'Stats',
+			title: 'Understand the Stats Page',
 			description:
-				'Your stats page includes a bunch of nifty graphs, charts, and lists that show you how many visits your site…',
+				'Your stats page includes a bunch of nifty graphs, charts, and lists that show you how many ' +
```
[the remark this answers is not in the document]
Comment (lsinger) [reply client/blocks/inline-help/contextual-help.js]: The ellipses were all generated by the API and should all have been removed.

File: client/blocks/inline-help/contextual-help.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ const fallbackLinks = [
 	},
 	{
 		link: 'https://en.support.wordpress.com/start/',
-		title: 'Get Started',
+		title: 'Five Steps to Getting Started',
```
[the remark this answers is not in the document]
Comment (lsinger) [reply client/blocks/inline-help/contextual-help.js]: Yes, it's a balance. We could go all generic and say "help article" for maximum maintainability! 😉 Just kidding, I think we're fine with having this level of specificity here.

File: client/blocks/inline-help/contextual-help.js [hunk 4]
```diff
@@ -40,129 +40,130 @@ const contextLinksForSection = {
...
-				'powers the user avatars on WordPress.com. Your WordPress.com and Gravatar ' +
-				'accounts are linked automatically. Once you configure...',
+				'powers the user avatars on WordPress.com.',
 		},
 		{
 			link: 'http://en.support.wordpress.com/account-deactivation/',
 			title: 'Account Deactivation',
-			description:
-				'If you are finished with your WordPress.com account and would like to shut it down, please follow the steps outlined...',
+			description: 'Finished with your WordPress.com account? Would you like to shut it down?',
```
[the remark this answers is not in the document]
Comment (lsinger) [reply client/blocks/inline-help/contextual-help.js]: Agree — on the fence.

**Comment by lsinger** [comment]
Thanks @spen — addressed your comments in 79406dc.

**Comment by michelleweber** [comment]
Okay! Some changes incoming. Broadly, I tried to refocus some of these to put the user in the prime, active position and get rid of the pass voice "our guide does X" constructions (the user does X, and we're support them). 

There's also a little general tightening, and a little more inspirational language in items where we want people to spend $ or explore advanced issues like child themes. Let me know if anything doesn't work for you/you need another revision!

Five Steps to Getting Started
> Quick-Start Guide

No matter what kind of site you want to build, our five-step checklist will get you set up and ready to publish.
> Use our five-step checklist to get set up and ready to publish, no matter what kind of site you're building.

Sometimes you need a fresh start. Learn how to deactivate your account.
> Need a fresh start? Learn how to deactivate your account.

Discover the WordPress.com Plans
> Explore the WordPress.com Plans

Upgrading to a plan unlocks a ton of features. Read more to understand which one would be the best fit for your needs.
> Upgrading your plan unlocks a ton of features! We'll help you pick the best fit for your needs and goals.

Learn how to use notifications to stay on top of all the things happening on WordPress.com.
> Notifications help you stay on top of the activity on your site _and_ all the things happening on WordPress.com -- learn how to use them.

Notifications from WordPress.com will be sent to the email address registered to your WordPress.com account. Learn how to manage them.
> WordPress.com sends email notifications to the email address registered to your account. Learn how to manage them.

When you leave a comment on a WordPress.com blog, you can choose to automatically receive email updates about other comments.
> When you leave a comment, you can automatically get email notifications for other new comments on the same post or page -- you'll never be out of the loop.

When you follow a blog on WordPress.com, the new posts from that site will appear in your Reader.
> When you follow a blog on WordPress.com, new posts from that site will automatically appear in your Reader.

In addition to built-in plugin functionality, WordPress.com Business allows you to install other plugins!
> Along with all the tools and features built right into WordPress.com, the Business plan allows you to install other plugins.

Need to upload your own plugins or themes? The Business plan allows you to do just that.
> When you want to build a one-of-a-kind website, it's time for WordPress.com Business: upload plugins and themes to create a truly tailored experience for your visitors. 

Set Up Your Blog in Five Steps
> Five Steps to Your Great New Blog

Our five-step checklist will get you set up and ready to publish to your blog.
> Get ready to publish? Our five-step checklist walks you through all the fundamentals.

Child themes allow you to extend your site beyond what’s possible with the existing tools. Learn how to use them!
> The only limit on your site is your vision -- if the themes you see don't match that, it's time to go beyond them. Learn to use child themes to customize and extend your website.

(For things like follow blogs/follow comments/add plugins, I would actually advocate for gerunds, rather than the regular verb and/or the repeated "how to, how to, how to.")

**Comment by spen** [comment]
Awesome effort @lsinger and thanks heaps @michelleweber for the detailed copy review - much appreciation to you both 💚 
Let's 🚢:) !

**Pull request merged**